### ISSUE TITLE
[Bug] Add space padding in logged message before writing exception message

### ISSUE THREAD

**Issue by maryamariyan** [dotnet/runtime#39531]
### Description of bug:

In the use case when we are also logging exception, there is a missing padding space between message and exception, when we use the single-lined default simple formatter.

### Expectation:

![image](https://user-images.githubusercontent.com/5897654/87812232-b5031280-c814-11ea-91f3-d5912fc95e9e.png)

### Actual
![image](https://user-images.githubusercontent.com/5897654/87798665-246f0700-c801-11ea-9f3e-a62e777e7e9d.png)

### Repro:
```c#
using var loggerFactory = LoggerFactory.Create(builder =>
    {
        builder.AddSimpleConsole(o =>
            {
                o.IncludeScopes = true;
                o.TimestampFormat = "HH:mm:ss ";
                o.SingleLine= true;
            });
    });

var logger = loggerFactory.CreateLogger<Program>();
using (logger.BeginScope(new { Message = "Hello" }))
using (logger.BeginScope(new KeyValuePair<string, string>("key", "value")))
using (logger.BeginScope("This is a scope message with number: {CustomNumber}", 11123))
{
    logger.LogInformation(exxx, "exception message with {0}", "stacktrace");
}
```

This commit should fix the problem: https://github.com/dotnet/runtime/commit/9027c78653d1ce8522b85e821b2d2e350c615f51

### Regression? 
No, Writing a colored single line log is a new feature, so this is not a regression.
This has a low risk bugfix and improves readability, so it would be good if we could submit this in preview 8.

<details>
 <summary> More Context about the different built-in Console Formatters (click to view) </summary>

We can use the built-in formatters in the following 4 different ways:

#### Json: 
Writes log in Json format.
```c#
builder.AddJsonConsole(o =>
{
    o.IncludeScopes = true;
    o.TimestampFormat = "HH:mm:ss ";
    o.JsonWriterOptions = new JsonWriterOptions() { Indented = true };
});
```
![image](https://user-images.githubusercontent.com/5897654/87815824-23e36a00-c81b-11ea-8977-02dff32ceda3.png)

#### Systemd: 
- Writes log in a single line with no colors
```c#
builder.AddSystemdConsole(o =>
{
    o.IncludeScopes = false;
    o.TimestampFormat = "HH:mm:ss";
});
```
![image](https://user-images.githubusercontent.com/5897654/87816143-b1bf5500-c81b-11ea-8c2f-be76a1b64740.png)

#### Simple: 
- Is the default formatter that can be configured to write with color or in a single line.
```c#
builder.AddSimpleConsole(o =>
{
    o.IncludeScopes = true;
    o.SingleLine= false;
    o.DisableColor= false;
    o.TimestampFormat = "HH:mm:ss ";
});
```
![image](https://user-images.githubusercontent.com/5897654/87799343-f0e0ac80-c801-11ea-9877-661fb76a0082.png)
</details>

**Comment by maryamariyan** [comment]
@ericstj This is a low risk bugfix and improves readability, so it would be good if we could submit this in preview 8.

**Comment by ericstj** [comment]
I believe this was fixed in #39534
